### ISSUE TITLE
Allow browsing of (cached) library when server cannot be reached

### ISSUE THREAD

**Issue by bobgautier** [abarisain/dmix#713]
It would be really nice to be able to browse my library content while I'm away from home at the CD store, so I can be sure I've not already got a CD before I buy it.   But when I tried that the other day, the app insisted on trying to connect to the server, and the library listing was tantalisingly out of reach behind the Cancel/Settings/Retry dialogue.

**Comment by hurzl** [comment]
thought about this, too ...

**Comment by abarisain** [comment]
Even though the library is displayed behind, MPDroid only knows whats
displayed, and tapping on an artist would not work for example. Even a
rotation could make you loose the list.

I'm not sure we can implement that easily (or want), can't you make your
server accessible from the internet ?

**Comment by hurzl** [comment]
Yes it yould only work with cache and it would have to display emptiness when the connection would be needed. So it's a bit complicated.
